pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -446,15 +448,12 @@ def list(self, request, *args, **kwargs):
         queryset = sorted(
             queryset, key=lambda obj: semantic_version.Version(obj.version), reverse=True
         )
-
         context = self.get_serializer_context()
-        page = self.paginate_queryset(queryset)
-        if page is not None:
-            serializer = self.get_serializer(page, many=True, context=context)
-            return self.get_paginated_response(serializer.data)
-
-        serializer = self.get_serializer(queryset, many=True, context=context)
-        return Response(serializer.data)
+        cvs_template_string = "[{{ versions|map('tojson')|join(',') }}]"
+        cvs_template = Template(cvs_template_string)
+        serialized_map = map(lambda x: self.get_serializer(x, context=context).data, queryset)
+        streamed = map(lambda x: x.encode("utf-8"), cvs_template.stream(versions=serialized_map))
+        return StreamingHttpResponse(streamed)
```

Comment:
should we take these to `UnpaginatedCollectionViewSet` also? Or the timeout only happens here?